stdlib/influxdata/influxdb/influxdb.flux

```diff
@@ -6,7 +6,14 @@
 package influxdb
 
 
-// cardinality returns the series cardinality of data stored in InfluxDB.
+// cardinality returns the series cardinality of data retrieved InfluxDB.
+//
+// Although this function is similar to InfluxQL's `SHOW SERIES CARDINALITY` it works in a slightly
+// different manner.
+//
+// cardinality is time bounded and reports the the cardinality of data that matches the conditions
+// passed into it rather than that of the bucket as a whole.
```

Comment:
```suggestion
// Cardinality is time bounded and reports the cardinality of the data that matches the conditions
// passed into it, rather than that of the bucket as a whole.
```